src/vs/workbench/workbench.web.main.ts

```diff
@@ -92,6 +92,7 @@ import { ITimerService, TimerService } from 'vs/workbench/services/timer/browser
 import { IDiagnosticsService, NullDiagnosticsService } from 'vs/platform/diagnostics/common/diagnostics';
 import { ILanguagePackService } from 'vs/platform/languagePacks/common/languagePacks';
 import { WebLanguagePacksService } from 'vs/platform/languagePacks/browser/languagePacks';
+import { IssueMainService } from 'vs/workbench/contrib/issue/browser/issueFormService';
```

Comment:
In environments that are not `electron-main`, I would not have `Main` in the service name. I introduced that only for declaring that a service is only available in `electron-main` layer.